platform/src/test/java/org/exoplatform/platform/qa/ui/platform/task/FilterTaskTestIT.java

```diff
@@ -0,0 +1,74 @@
+package org.exoplatform.platform.qa.ui.platform.task;
+
+import com.codeborne.selenide.Condition;
+import com.codeborne.selenide.Configuration;
+import org.exoplatform.platform.qa.ui.commons.Base;
+import org.exoplatform.platform.qa.ui.core.PLFData;
+import org.exoplatform.platform.qa.ui.selenium.platform.HomePagePlatform;
+import org.exoplatform.platform.qa.ui.selenium.platform.ManageLogInOut;
+import org.exoplatform.platform.qa.ui.task.pageobject.ProjectsManagement;
+import org.exoplatform.platform.qa.ui.task.pageobject.TasksManagement;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static com.codeborne.selenide.Selectors.byText;
+import static com.codeborne.selenide.Selenide.$;
+import static org.exoplatform.platform.qa.ui.selenium.Utils.getRandomNumber;
+import static org.exoplatform.platform.qa.ui.selenium.locator.taskmanagement.TaskManagementLocator.*;
+import static org.exoplatform.platform.qa.ui.selenium.logger.Logger.info;
+
+/**
+ * Created by esmaoui on 27/02/2018.
+ */
+@Tag("sniff")
+@Tag("task")
+public class FilterTaskTestIT extends Base {
+    HomePagePlatform     homePagePlatform;
+    ManageLogInOut       manageLogInOut;
+    TasksManagement      tasksManagement;
+    ProjectsManagement   projectsManagement;
+
+
+    @BeforeEach
+    public void setupBeforeMethod() {
+        info("Start setUpBeforeMethod");
+        homePagePlatform = new HomePagePlatform(this);
+        tasksManagement = new TasksManagement(this);
+        projectsManagement = new ProjectsManagement(this);
+        manageLogInOut = new ManageLogInOut(this);
+        manageLogInOut.signInCas(PLFData.USER_ROOT,PLFData.password);
+
+    }
+
+    @Tag("TA-571")
```

Comment:
Do we add a tag with the issue number now for each issue ?

Reply:
Yes to know which issue reported has been automated.

Reply:
Can you show me how you use this tag ? Is there a report somewhere ?

Reply:
Yes we just mention the issue number to the test case, so we have a direct link from the code to the origin of the automated case so we can have a reference as specification for later re-use of question about from where we specified the expected behavior.
In addition of setting the PR on the issue to have a link from both sides.